sdk/storage/src/blob/blob/mod.rs

```diff
@@ -300,8 +311,8 @@ impl Blob {
                 content_type,
                 content_encoding,
                 content_language,
-                content_md5,
-                content_crc64,
+                content_md5: content_md5.map(ConsistencyMD5),
+                content_crc64: None, // TODO
```

Comment:
Hmmm... if this really is a todo we should probably make an issue for it. I'm not sure I fully understand why this is a todo though. Can you explain more?

Reply:
I have no special explanation except that the is a field in the structure which could be `Some`, but with what? The Azure API documentation does not mention any CRC64 header. Maybe another system?